docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -2,9 +2,8 @@
 //
 // * docs/web-console-guide/master.adoc
 
-:_content-type: PROCEDURE
+:_mod-docs-content-type: PROCEDURE
 [id="mta-7-installing-web-console-on-openshift_{context}"]
```

Comment:
id="mta-7-installing-web-console-on-openshift_{context}"]
This also needs to be changed to 7.1 right ?

Reply:
@sshveta No, at this point, the same file is used for all the 7.x installation instructions, as opposed to the 6.x instructions.